compliance/aws/ecs_unused/aws_unused_ecs_clusters.pt

```diff
@@ -209,17 +211,24 @@ end
 policy "pol_ri_coverage" do
   validate $ds_unused_ecs_cluster_map do
     summary_template "{{ rs_project_name }} (Account ID: {{ rs_project_id }}): {{ len data }} Unused ECS Clusters found in AWS."
-    detail_template <<-EOS
-# Unused ECS Clusters
-| Cluster Name | Region | status | Tags |
-| ------------ | ------ | ------ | ---- |
-{{ range data -}}
-| {{ .cluster_Name}} | {{.region}} | {{.status}} | {{.tagKeyValue}} |
-{{ end -}}
-    EOS
     escalate $report_unused_ECS_clusters
     escalate $delete_clusters
     check eq(size(data),0)
+    export do
+      resource_level true
+      field "id" do
+        label "Cluster Name"
+      end
+      field "region" do
+        label "Region"
+      end
+      field "status" do
+        label "Status"
+      end
+      field "Tags" do
+        label "tagKeyValue"
```

Comment:
label should be Tags, field should be tagKeyValue

Reply:
Done!